src/main.rs

```diff
@@ -723,6 +725,8 @@ fn start_vmm(cmd_arguments: ArgMatches) -> Result<Option<String>, Error> {
                     config::RestoreConfig::parse(restore_params).map_err(Error::ParsingRestore)?,
                 )
                 .map_err(Error::VmRestore)?;
+        } else {
+            return Err(Error::MissingPayload);
```

Comment:
This behavior change does not look right to me. It is perfectly legit to start a VMM without any payload or any VM configurations, say starting a VMM with a API socket only.